### PR TITLE
I18n: Make preview popup loader strings translateable

### PR DESCRIPTION
**Pull request by Soean**
## Description
Adds translate functions to the text strings in the preview popup loader.

### CODE REVIEW

File: editor/header/preview-button/index.js [hunk 1]
```diff
@@ -76,8 +76,8 @@ export class PreviewButton extends Component {
 
 		const markup = `
 			<div>
-				<p>Please wait&hellip;</p>
-				<p>Generating preview.</p>
+				<p>${ __( 'Please wait&hellip;' ) }</p>
```
Comment (aduth) [inline editor/header/preview-button/index.js]: Unlike with React, we don't get string escaping for free here, so there is a potential risk with inserting this into raw markup.

https://codex.wordpress.org/I18n_for_WordPress_Developers#HTML

We may need to consider patterns for sanitization.

Comment (aduth) [reply editor/header/preview-button/index.js]: Related: https://github.com/WordPress/packages/pull/12

Comment (aduth) [reply editor/header/preview-button/index.js]: We could also generate this with `wp.element.renderToStaticMarkup` as an element.

Or just [Lodash `_.escape`](https://lodash.com/docs/4.17.5#escape)

**Review by aduth — CHANGES_REQUESTED** [review]
@Soean Are you able to revisit this one? Should we just close it?

**Comment by tofumatt** [comment]
Looks like this PR has got stale, so closing it for now. If you can revisit it feel free to re-open this PR or open a new PR.

**Comment by tofumatt** [comment]
Closed this but filed #7725 to track this issue.

**Pull request closed without merging**